src/Events/Custom_Tables/V1/Activation.php

```diff
@@ -48,41 +48,41 @@ public static function activate() {
 	 * @since 6.0.0
 	 */
 	public static function init() {
+		$services = tribe();
+
 		// Check if we ran recently.
 		$db_hash = get_transient( static::ACTIVATION_TRANSIENT );
 
-		$schema_builder = tribe( Schema_Builder::class );
+		$schema_builder = $services->make( Schema_Builder::class );
 		$hash           = $schema_builder->get_registered_schemas_version_hash();
 
-		if ( $db_hash == $hash ) {
-			return;
+		// Sync any schema changes we may have.
+		if ( ! $schema_builder->all_tables_exist( 'tec' ) ) {
```

Comment:
Just want a sanity check, as I recall we moved this under the transient for performance concerns, maybe it was added after a ticket for performance... @lucatume are you comfy with this check running often?

Reply:
I will run some tests and work on figuring that out a little more. I would love to avoid having to run that method all the time.